specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-01-14-preview/desktopvirtualization.json

```diff
@@ -2467,6 +2509,54 @@
               "type": "string"
             }
           }
+        },
+        "isDataAction": {
+          "description": "Is a data action.",
+          "type": "boolean"
+        },
+        "properties": {
+          "$ref": "#/definitions/OperationProperties"
+        }
+      }
+    },
+    "OperationProperties": {
```

Comment:
In the model definition, it seems some of them are using pascal case while others are using camel case, could you make it to be consisitent?